mutant/mutant1.md

```diff
@@ -20,7 +20,7 @@ There are two rake tasks to help you:
 
 [Mutant](https://github.com/mbj/mutant#readme) can also be ran against individual methods:
 
-    bundle exec mutant --rspec-full -r ./spec/document_spec.rb ::SecureDB::Document#grant_access
+    bundle exec mutant -I lib/ -r secure_db --use rspec "SecureDB::Document#grant_access"
```

Comment:
Just a small nitpick, but I usually use long form arguments for any code I show to others as these are better self explaning. Only short forms for local CLI, hence I recommend `--include` over `-I`.